src/Build/Dockerfiles/Dockerfile.base

```diff
@@ -0,0 +1,14 @@
+# Define an image to start from. 
+FROM python:3.10.0-slim
+
+# Set the working directory within the container
+WORKDIR /workspace
+
+# Install the dependencies for the gradio app
+RUN python -m pip install pyyaml gradio==3.25.0 gradio_client==0.1.3 huggingface-hub==0.14.0 pydantic==1.10.7
+
+# Add the files to the container
+ADD ../parser.py /workspace/
+
+# Define the command to run the app
+CMD ["python", "parser.py"]
```

Comment:
There's two ways we can go. Either we generate `app.py` outside of any docker images, and then copy it in here, OR we copy in the entire algorithm we end up wrapping, and run the generator in this docker.

Remember this will all be done in CI/CD, so it will be run in clean VMs.

Reply:
As discussed later, We are now moving with just adding `app.py` from the Base-Dockerfile to the Docker-Workspace (here `bilayers`) and in the noxfile.py, we are mentioning the steps to generate `app.py` by running `parse.py` and `generate.py`. Hence, `app.py` would be generated outside and we 'll just add that in the docker.